.env.example

```
@@ -12,4 +12,5 @@ NEXT_PUBLIC_FIREBASE_PROJECT_ID=notakto-3359b
 NEXT_PUBLIC_FIREBASE_STORAGE_BUCKET=notakto-3359b.firebasestorage.app
 NEXT_PUBLIC_FIREBASE_MESSAGING_SENDER_ID=467221757682
 NEXT_PUBLIC_FIREBASE_APP_ID=1:467221757682:web:9d896324f25e6dce03da6b
-NEXT_PUBLIC_FIREBASE_MEASUREMENT_ID=G-MLBPJWB1C2
\ No newline at end of file
+NEXT_PUBLIC_FIREBASE_MEASUREMENT_ID=G-MLBPJWB1C2
+NEXT_PUBLIC_API_URL=https://notakto-solo-1.onrender.com/v1
```

dotenv-linter (4.0.0)

[warning] [EndingBlankLine] No blank line at the end of the file

(EndingBlankLine)